lib/cog/subcommand.rb

```diff
@@ -1,16 +1,3 @@
 class Cog
-  class SubCommand
-
-    def initialize(request = nil)
-      @request = request
-    end
-
-    private
-
-    def request
-      @request
-    end
-
-  end
+  class SubCommand < Cog::Command; end
```

Comment:
Should we just drop the subcommand class and use command instead? Wouldn't be too tough to change the few subcommands we've written so far.